go.mod

```diff
@@ -29,7 +29,7 @@ require (
 	gomodules.xyz/jsonpatch/v2 v2.2.0
 	gopkg.in/square/go-jose.v2 v2.6.0
 	k8s.io/api v0.25.8
-	k8s.io/apimachinery v0.27.0-beta.0
```

Comment:
How did we end up with this version ? 😓

Reply:
It got pulled in from https://github.com/jenkins-x/go-scm/blob/ae6a81424dcc7b0742505c12a823255f02bc2eb6/go.mod#L17 via https://github.com/tektoncd/pipeline/pull/6394

We'd either need to fix the version upstream in go-scm + pull in the new version or downgrade go-scm (not sure how long we'd want to keep this downgraded for).

Alternatively, we only use go-scm in the git resolver, which I'm not 100% sure we need in the main pipeline repo. We could consider breaking this out into its own repo + module if needed.